string Command
int16 EyeIndex
---
int16 Value
